txbits/conf/evolutions/default/3.sql

```diff
@@ -185,7 +185,7 @@ begin
   where id = a_id and address = a_address and
   tx_hash = a_tx_hash and confirmed is NULL;;
 
-  if found and d.user_id <> 0 then
+  if found then
```

Comment:
It was a bug. You can see there's a check for `if d.user_id <> 0` 5 lines later within this block. The bug was discovered upon writing a unit test that simulates the hot wallet being refilled.